src/Controller/EventController/Handler/CreateEventHandler/CreateEventHandlerInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Controller\EventController\Handler\CreateEventHandler;
+
+interface CreateEventHandlerInterface
+{
+
+}
```

Comment:
Empty string to end of line
```suggestion
}

```